src/v2/guide/instance.md

```diff
@@ -67,37 +69,40 @@ var vm = new Vue({
 vm.$data === data // -> true
 vm.$el === document.getElementById('example') // -> true
 
-// $watch is an instance method
+// $watch est une méthode de l'instance
 vm.$watch('a', function (newVal, oldVal) {
-  // this callback will be called when `vm.a` changes
+  // cette fonction de retour sera appellée quand `vm.a` changera
 })
 ```
 
-<p class="tip">Don't use [arrow functions](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) on an instance property or callback (e.g. `vm.$watch('a', newVal => this.myMethod())`). As arrow functions are bound to the parent context, `this` will not be the Vue instance as you'd expect and `this.myMethod` will be undefined.</p>
+<p class="tip">
+N'utilisez pas les [fonctions fléchées](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Fonctions/Fonctions_fl%C3%A9ch%C3%A9es) sur une propriété ou fonction de retour d'une instance  (par exemple `vm.$watch('a', newVal => this.myMethod())`). Comme les fonctions fléchées sont liées au contexte parent, `this` ne sera pas l'instance de Vue comme vous pourriez vous y attendre et `this.myMethod` sera indéfini.
+</p>
 
-Consult the [API reference](../api) for the full list of instance properties and methods.
+Consultez [l'API](../api) pour une liste complète des propriétés et méthodes d'une instance. 
 
-## Instance Lifecycle Hooks
+## Les hooks de cycles de vie d'une instance
 
-Each Vue instance goes through a series of initialization steps when it is created - for example, it needs to set up data observation, compile the template, mount the instance to the DOM, and update the DOM when data changes. Along the way, it will also invoke some **lifecycle hooks**, which give us the opportunity to execute custom logic. For example, the `created` hook is called after the instance is created:
+Chaque instance de vue traverse une série d'étapes d'initialisations au moment de sa création - par exemple, elle doit mettre en place l'observation des data, compiler le template, monter l'instance sur le DOM et mettre à jour le DOM quand les data changent. En cours de route, elle va aussi invoquer des **hooks de cycles de vie**, qui nous donnent l'opportunité d'exécuter de la logique custom. Par exemple, le hook `created` est appelé après que l'instance sera créee.
 
 ``` js
 var vm = new Vue({
   data: {
     a: 1
   },
   created: function () {
-    // `this` points to the vm instance
+    // `this` référence l'instance de vm
```

Comment:
Pas compris, pour moi c'est le verbe référencer, donc this "référence" l'instance. Non ?